agentMET4FOF/metrological_streams.py

```diff
@@ -60,7 +47,7 @@ class MetrologicalSineGenerator(MetrologicalDataStreamMET4FOF):
     to be supplied to the `set_generator_function`
 
     """
-    def __init__(self,sfreq = 500, F=5):
+    def __init__(self, sfreq = 500, F=5):
         super().__init__()
         self.set_metadata("SineGenerator","time","s",("Voltage"),("V"),"Simple sine wave generator")
```

Comment:
Instead of providing rather randomly chosen default metadata, I would suggest to require the user to specify the metadata on intialization and then provide a tutorial on how to do this (tutorial 4).

Reply:
Do you mean something like a text input-field on the dashboard?

Reply:
:open_mouth: Uh, I think, that would be awesome, but actually I was just talking about input parameters in the code during the intialization of a metrological agent.

Reply:
The second thing can be done pretty easily, I think. The dashboard solution is a bit more ambitious.

Reply:
That's what I thought...